examples/README.md

```diff
@@ -1,15 +1,67 @@
-## Contribution Guidelines
+<p align="center">
+  <a href="https://www.immutable.com">
+    <img src="https://cdn.dribbble.com/users/1299339/screenshots/7133657/media/837237d447d36581ebd59ec36d30daea.gif" width="280"/>
+  </a>
 
-Important information on how to create examples, pull them through to the docs site and ensure they are covered by tests in the CI CD pipeline.
+  <h2 align="center">Immutable Code Examples</h3>
 
+  <p align="center">
+    Immutable's code examples as used for the code snippets in the Immutable Docs
+    <br />
+    <a href="https://docs.immutable.com"><strong>Explore the docs »</strong></a>
+    <br />
+  </p>
+</p>
+<hr />
 
-## Example Scope
+**Table of Contents**
 
-When creating an example app it should contain only examples about one particular feature to prevent overloading the example. If there are multiple ways to use the feature then it is okay to include those in one sample app.
+- [Introduction](#introduction)
+- [Running examples locally](#running-examples-locally)
+- [Contribution guidelines](#contribution-guidelines)
+- [Adding examples](#adding-examples)
+- [End to end testing](#end-to-end-testing)
+- [Using code examples in the docs site](#using-code-examples-in-the-docs-site)
+<hr />
 
-For example;
+# Introduction
 
-the app in `examples/passport/wallets-connect-with-nextjs` show how to connect passport in the nextjs framework. It contains a default route that has links to each of the examples. Inside there are three routes, one for each way to connect (EIP-1193, EtherJS and Wagmi). These are okay to be part of one sample app since they show how to use one feature but using 3 different libraries.
+The example apps in this examples directory are compiled and tested as part of our CI CD pipeline. The goal of this is to ensure the examples found here are always able to run against the latest version of the Immutable Typescript SDK. 
+
+Selected portions of code from these example apps is then displayed as code snippets in our docs site, which means our code snippets in the docs are inherently always accurate as well. How to include the code from here in the docs site is explained below. 
+
+These example apps can also be used directly as a reference and run locally to test and develop with.
+
+# Running examples locally
+
+First you need to clone the `ts-immutable-sdk` repo. There is no need to install the root node modules or build the SDK since the examples have their own `package.json` files and pull the node modules from npm rather than the local build.
```

Comment:
all good. i'll re word this. the reason why i thought that was probably because i alraeady had the other dependencies installed so it was fast, but also when i run `yarn install` it seems to create node_modules folders inside the identity-with-nextjs and wallets-transactions-with-nextjs which i would not have expected if they were being installed to the root.

Reply:
I think it will still create node_modules to the apps if those dependencies are used only in that app and no where else